components/portal/src/components/common/ImageList.js

```diff
@@ -119,21 +120,40 @@ class ImageList extends React.Component {
-                                                    <Typography variant={"caption"} className={classes.block}
-                                                        color={"textPrimary"}>
-                                                        <AccessTime className={classes.updated}/> Last Updated on&nbsp;
-                                                        {
-                                                            moment(image.updatedTimestamp)
-                                                                .format(Constants.Format.DATE_TIME)
-                                                        }
-                                                    </Typography>
+                                                    {
+                                                        image.updatedTimestamp
+                                                            ? (
+                                                                <Typography variant={"caption"} color={"textPrimary"}
+                                                                    className={classes.block}>
+                                                                    <AccessTime className={classes.updated}/>
+                                                                              &nbsp;Last Updated on&nbsp;
+                                                                    {
+                                                                        moment(image.updatedTimestamp)
+                                                                            .format(Constants.Format.DATE_TIME)
+                                                                    }
+                                                                </Typography>
+                                                            )
+                                                            : <Typography variant={"caption"} color={"textPrimary"}
+                                                                className={classes.block}>
+                                                                <InfoOutlined className={classes.updated}/>
+                                                                          &nbsp;No versions available
+                                                            </Typography>
+                                                    }
                                                 </React.Fragment>
                                             }/>
-                                        <GetApp className={classes.elementIcon}/>
-                                        <Typography variant={"subtitle2"} color={"inherit"}
-                                            className={classes.elementText}>
-                                            {image.pullCount}
-                                        </Typography>
+                                        {
+                                            image.pullCount === null
```

Comment:
Shall we make this `!image.pullCount && image.pullCount !== 0` ?

Reply:
Shouldn't we show the pull count 0 for images that are never pulled?